src/app/core.ts

```diff
@@ -1,9 +1,32 @@
 import {asperaDesktop} from '../index';
-import {client} from '../helpers/client';
-import {errorLog, generateErrorBody, generatePromiseObjects, getWebsocketUrl, isValidTransferSpec, randomUUID, throwError} from '../helpers/helpers';
 import {messages} from '../constants/messages';
+import {
+  errorLog,
+  generateErrorBody,
+  generatePromiseObjects,
+  getWebsocketUrl,
+  isSafari,
+  isValidTransferSpec,
+  randomUUID,
+  throwError
+} from '../helpers/helpers';
+import {httpClient} from '../helpers/client/http-client';
+import {safariClient} from '../helpers/client/safari-client';
```

Comment:
You could re-export both of these from `client` module to clean this up a bit. Then it would be:
```suggestion
import {httpClient, safariClient} from '../helpers/client';
```

Reply:
Done!